src/utils/api/setAccountInfo.ts

```diff
@@ -25,17 +25,20 @@ interface ISetAccountInfoPayload extends IAPICall {
   };
 }
 
-async function setAccountInfo(secret: string, accountName: string, accountDescr: string) {
+async function setAccountInfo({ ...args }) {
```

Comment:
can this be defined as `{ ...args }: ISetAccountInfo` so it has a type?

Reply:
yes, done